consensus/byzantine_test.go

```diff
@@ -29,7 +26,10 @@ func init() {
 func TestByzantine(t *testing.T) {
 	N := 4
 	logger := consensusLogger().With("test", "byzantine")
-	css := randConsensusNet(N, "consensus_byzantine_test", newMockTickerFunc(false), newCounter)
+	css, rootDirs := randConsensusNet(N, "consensus_byzantine_test", newMockTickerFunc(false), newCounter)
+	for _, dir := range rootDirs {
```

Comment:
```suggestion
defer func() {
  for _, dir := range configRootDirs {
    _ = os.RemoveAll()
  }
}()
```

Reply:
How's this better than deferring `os.RemoveAll()` kinda atomic calls individually?

Reply:
smaller defer stack? faster?

Reply:
done